### DISCUSSION TITLE
Dashboard displayed on a tablet

### DISCUSSION THREAD

**Discussion by abd7786** [jasonacox/Powerwall-Dashboard#361] [General]
Just managed to get the dashboard setup on windows 11 successfully, thanks to the instructions here.

Wondering if anyone uses a 10" tablet to display the dashboard? 

If so:
1. Do you use it in portrait or landscape mode?
2.  Is the text easily legible or is the tablet screen too small to display it well enough?
3. How did you mount it?
4. How do you provide power to it?

I have a PoE switch and a poe to usb c injector, so that should work.

**Comment by jasonacox** [comment]
I haven't seen that yet. If you (or anyone else) manages to get that working, please post pictures! 😄

**Comment by SCHibbard** [comment]
Catching up on old threads.  I use the dashboard frequently on an iPad. Usually use landscape, text is fine.  Have not wall mounted one, so can't answer those questions (though PoE is the best if you can find a tablet that can be powered off it). Grafana has a mind of it's own in deciding how to arrange & scale panels on a different screen size.  I had two issues:
1) Panels next to each other on a usual screen size may get rearranged, and not necessarily in the order you would like them.
2) Instead of scaling down some panels, Grafana will sometimes go to scrolling, in particular legends - even when there is plenty of space!

My solution was to create a separate dashboard where I fiddled with positioning and size of the panels to look nice on the iPad.
